cmd/gpu-feature-discovery/main.go

```diff
@@ -86,6 +86,12 @@ func main() {
 			Value:   "/etc/kubernetes/node-feature-discovery/features.d/gfd",
 			EnvVars: []string{"GFD_OUTPUT_FILE"},
 		},
+		&cli.StringFlag{
+			Name:    "imex-nodes-config",
+			Usage:   "the name of the nvidia-imex nodes config file",
+			Value:   "nodes_config.cfg",
```

Comment:
@klueska default value now set

Reply:
I realize I've gone back and forth on this, but I think this should be a full path now, to remain consistent with the other options, e.g. `output-file` (whose prefixes are also something mounted into the container).